gradle/src/main/groovy/com/flurry/android/symbols/NdkSymbolUpload.groovy

```diff
@@ -48,15 +48,20 @@ class NdkSymbolUpload {
             logger.lifecycle("Found .so file: " + sharedObjectFile.parentFile.name + "/" + sharedObjectFile.name)
             files.add(sharedObjectFile.absolutePath)
         }
-        
-        Collection<ExternalNativeBuildTask> tasks = variant.externalNativeBuildTasks
-        for (ExternalNativeBuildTask task : tasks) {
+
+        Closure searchForSharedObjectFiles = { ExternalNativeBuildTask task ->
             File objFolder = task.objFolder
             File soFolder = task.soFolder
             findSharedObjectFiles(objFolder, addSharedObjectFiles)
             findSharedObjectFiles(soFolder, addSharedObjectFiles)
         }
 
+        try {
+            variant.getExternalNativeBuildProviders().each { taskProvider -> searchForSharedObjectFiles(taskProvider.get()) }
+        } catch(Throwable ignored) {
```

Comment:
Why are we doing the same thing in the `catch` block? What is this even catching? If this is to maintain compat for older versions let's document that, and maybe catch something more specific like `MethodNotFound` or whatever is actually thrown. Ditto all the other `try/catch` constructs like this